### PR TITLE
Add shutdown API specs

### PR DESCRIPTION
**Pull request by swallez**
As titled

### CODE REVIEW

**Review by delvedor — CHANGES_REQUESTED** [review]
LGTM, please run `make contrib`

**Comment by swallez** [comment]
Ran `make contrib`. Please cross-check!

File: specification/shutdown/delete_node/ShutdownDeleteNodeResponse.ts [hunk 1]
```diff
@@ -18,5 +18,7 @@
  */
 
 export class Response {
```
Comment (delvedor) [inline specification/shutdown/delete_node/ShutdownDeleteNodeResponse.ts:20]: You should use `AcknowledgedResponseBase` :)
```suggestion
export class Response extends AcknowledgedResponseBase {}
```

File: specification/shutdown/put_node/ShutdownPutNodeResponse.ts [hunk 2]
```diff
@@ -18,5 +18,7 @@
  */
 
 export class Response {
```
Comment (delvedor) [inline specification/shutdown/put_node/ShutdownPutNodeResponse.ts:20]: Same here.

**Review by delvedor — CHANGES_REQUESTED** [review]
Just a nit.

**Comment by swallez** [comment]
Fixed to use `AcknowledgedResponseBase` (this actually more than an nit!)

**Review by delvedor — APPROVED** [review]

**Pull request merged**